src/components/Common/UI/Badge/BadgeTypes.ts

```diff
@@ -0,0 +1,9 @@
+import type { HTMLAttributes } from 'react'
+
+export interface BadgeBaseProps
+  extends Pick<HTMLAttributes<HTMLSpanElement>, 'className' | 'id' | 'aria-label'> {
+  text: string
+  variant?: 'success' | 'warning' | 'error' | 'info'
```

Comment:
Can we use `status` here as a name instead to keep this naming aligned with Alert? Maybe it's worth getting a shared type between these?